app/helpers/home_helper.rb

```diff
@@ -18,9 +22,13 @@ def actioned
 
     def generate_actioned
       scope, actioned = Petition.visible, {}
+      awaiting_response = scope.awaiting_response.awaiting_response.by_waiting_for_response_longest
```

Comment:
I think you've got an extra awaiting_response scope here? Not sure if that was intentional.

```
scope.awaiting_response.awaiting_response.by_waiting_for_response_longest
should be
scope.awaiting_response.by_waiting_for_response_longest
```